site/guides/enhanced-apis/token.md

```diff
@@ -0,0 +1,113 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: How to Fetch a Smart Account's ERC-20 Tokens
+  - - meta
+    - name: description
+      content: Follow this guide to fetch a Smart Account's ERC-20 Tokens with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
+  - - meta
+    - property: og:description
+      content: Follow this guide to fetch a Smart Account's ERC-20 Tokens with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
+  - - meta
+    - name: twitter:title
+      content: How to Fetch a Smart Account's ERC-20 Tokens
+  - - meta
+    - name: twitter:description
+      content: Follow this guide to fetch a Smart Account's ERC-20 Tokens with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
+---
+
+# How to Fetch a Smart Account's ERC-20 Tokens
+
+Alchemy provides several [Enhanced APIs](https://www.alchemy.com/enhanced-apis), which are especially useful for querying information about the smart accounts you create using Account Kit, such as the account's ERC-20 Token balances using the [Token API](https://www.alchemy.com/token-api).
+
+There are three ways you can use Alchemy's Token API using Account Kit:
+
+1. via extending the Alchemy Provider [with Enhanced APIs](/packages/aa-alchemy/provider/withAlchemyEnhancedApis.md)
+2. via the [Alchemy SDK](https://www.alchemy.com/sdk) directly
+3. via a custom [Viem Client](https://viem.sh/docs/clients/custom.html)
```

Comment:
is it necessary to include the doc/examples for these two? This seems out of scope for Account Kit doc, making it complicated. Would put these in a separate doc section (Using Enhanced API vs On your own)